simpleocr/improver.py

```diff
@@ -36,6 +36,7 @@ def enhance_image(imagefile, color=None, brightness=None, contrast=None, sharpne
     imagefile.image = pillow_to_numpy(image)
     return imagefile
 
+
```

Comment:
This is a PEP-8 violation fix automatically performed by PyCharm. If you don't want it to stay @goncalopp , I will remove it and it could be looked at some other time or left like this.